webserver/views/api.py

```diff
@@ -37,7 +38,13 @@ def get_low_level(mbid):
     try:
         return Response(load_low_level(mbid, offset), content_type='application/json')
     except NoDataFoundException:
-        raise NotFound
+        raise webserver.exceptions.InvalidAPIUsage("Not found", status_code=NotFound.code)
+
+@api_bp.route("/<string:notanmbid>/low-level", methods=["GET"])
+@crossdomain()
+def get_low_level_404(notanmbid):
+    """Specific low-level view for non-mbid url parts to return a json error message"""
+    raise webserver.exceptions.InvalidAPIUsage("Not found", status_code=NotFound.code)
```

Comment:
The thing is, I don't want to get an html error message for the potential case of a non-uuid argument. 
I don't mind this setup, but I would be just as happy with a non-specific uuid matcher and do validation inside the method (return 404 if it's not a valid uuid)